testapi.pm

```diff
@@ -433,6 +433,29 @@ sub autoinst_url() {
     return "http://10.0.2.2:" . (get_var("QEMUPORT") + 1);
 }
 
+=head2 data_url
+
+data_url($name)
+
+returns the URL to download data or asset file
+Special values REPO_\d and ASSET_\d points to the asset configured
+in the corresponding variable
+
+=cut
+
+sub data_url($) {
+    my ($name) = @_;
+    if ($name =~ /^REPO_\d$/) {
+        return autoinst_url . "/assets/repo/" . get_var($name);
+    }
```

Comment:
I'm not so sure this is what we want in the long run. http installations are nice, but far from all installation ways we support. So where would you call that in a test?